### DISCUSSION TITLE
OpenVPN connection problem

### DISCUSSION THREAD

**Discussion by kestouf** [haugene/docker-transmission-openvpn#2359] [Q&A]
Hello
I'm sorry to come to you, but after hours of searching to solve this problem, I haven't been able to solve it. Mon VPN est [[cyberghost]](https://www.cyberghostvpn.com/fr_FR/)

I'm trying to install docker-transmission-openvpn but I'm having a problem launching docker with openVPN.
Here is the error message: https://pastebin.com/1i0pFtpF
Here my docker-compose (The PID is that of a user with admin rights) : https://pastebin.com/TPuX3Ndf

I see that it finds the default.ovpn file, but it does not find the other files (ca.crt client.crt client.key) yet they are all in the same folder: /volume1/Informatique/CONFDOCKER/transmissionVPN/VPN

Here is the total text of the docker startup: https://pastebin.com/SZXWUDQz

Can you help me please ?

Another question. Do you have a link to a tutorial that explains step by step the installation and configuration of this docker image?

Thanks

**Comment by pkishino** [comment]
whats the reason for using a custom ovpn file?
https://haugene.github.io/docker-transmission-openvpn/ is the documentation
most likely your custom ovpn file is pointing at the wrong directories for the crt and key files.. please share this ovpn file
